advocacy_docs/edb-postgres-ai/migration-etl/index.mdx

```diff
@@ -9,34 +9,56 @@ navigation:
 
-The AI copilot is designed to be user-friendly and easy to use. Users can interact with the AI copilot using natural language and improve the quality of answers with [good prompting](/migration_portal/latest/03_mp_using_portal/mp_ai_copilot/ai_good_prompts/). The AI copilot is also designed to be context-aware, so it can provide users with relevant information based on the context of the conversation.
+## EDB Postgres AI Migration Portal
 
-## The EDB Postgres AI Replication Server
+[EDB Postgres AI Migration Portal](/migration_portal/latest/) is a web-based tool that helps you assess and analyze Oracle database schemas and convert them to EDB Postgres Advanced Server.
 
-The [EDB Postgres Replication Server](/eprs/latest/) is a tool that helps you keep your Postgres database up-to-date with changes from other databases.
+EDB Postgres® AI Migration Portal offers an [AI copilot](/migration_portal/latest/03_mp_using_portal/mp_ai_copilot/) to assist users who are migrating their databases to EDB Postgres Advanced Server.
 
-## The EDB Postgres AI Migration Toolkit
+- **Purpose:** Conversion and import of schemas
 
-The [EDB Postgres Migration Toolkit](/migration_toolkit/latest/) is a set of tools that help you migrate your data from other databases to Postgres.
+- **Applicable migration scenarios:** From Oracle to EDB Postgres Advanced Server
+
+## EDB Postgres AI Migration Toolkit
+
+[EDB Postgres Migration Toolkit](/migration_toolkit/latest/) is a set of tools that help you migrate database objects and data at a granular level from non-EDB databases to EDB Postgres.
+
+- **Purpose:** Migration of database objects and data, includes options for more granular control of the process.
+
+- **Applicable migration scenarios:** From Oracle, MySQL, Microsoft SQL Server, Sybase Adaptive Server Enterprise or Postgresql to EDB Postgres Advanced Server or PostgreSQL.
+
+## EDB Oracle Migration Handbook 
+
+The Oracle Migration Handbook is not a tool or software, but a set of guidelines to help you through your migration journey.
+
+The purpose of the [Migrantion Handbook](/migrating/oracle/) is to help you plan and execute an Oracle to EDB Postgres Advanced Server migration. It identifies and describes the factors to consider when performing a migration, the steps in the migration journey, and the capabilities, tools, and services that EDB provides to help with migrations. It also covers the key similarities and differences between Oracle and EDB Postgres Advanced Server and workarounds for known differences or incompatibilities.
+
+- **Purpose:** Planning and walking you through a migration from Oracle.
+
+- **Applicable migration scenarios:** From Oracle to EDB Postgres Advanced Server
+
+## EDB Postgres AI Replication Server
```

Comment:
If you want to be comprehensive, I would also add a section for LiveCompare at a minimum.